ui/packages/ui/src/app/pages/createConnector/federatedModule/DataOptions.tsx

```diff
@@ -39,35 +39,48 @@ const checkIfRequired = (
 ): boolean => {
   const matchProp = _.find(
     propertyList,
-    (obj) => obj.name === property.replace(/[_]/g, ".")
+    (obj) => obj.name === property
   );
   return matchProp ? matchProp.isMandatory : false;
 };
 
+const getMappingGeneralProperty = (
+  propertyList: ConnectorProperty[]
+): ConnectorProperty[] => {
+  const propertyDefinitionsCopy = _.cloneDeep(propertyList);
+  return propertyDefinitionsCopy.filter(
+    (defn: any) => defn.category === PropertyCategory.DATA_OPTIONS_GENERAL
+  );
+};
+
+const getMappingAdvanceProperty = (
+  propertyList: ConnectorProperty[]
+): ConnectorProperty[] => {
+  const propertyDefinitionsCopy = _.cloneDeep(propertyList);
+  return propertyDefinitionsCopy.filter(
+    (defn: any) => defn.category === PropertyCategory.DATA_OPTIONS_GENERAL
```

Comment:
This one should be PropertyCategory.DATA_OPTIONS_ADVANCED

Reply:
done.